### ISSUE TITLE
Infinite loop for numericInput mask

### ISSUE THREAD

**Issue by Gyrbo** [RobinHerbots/Inputmask#128]
The following jsFiddle demonstrates the problem: http://jsfiddle.net/SRTJj/

There is an infinite loop on line 967:

```
while (getBufferElement(bfrClone, firstUnmaskedPosition, true) != getPlaceHolder(firstUnmaskedPosition) && firstUnmaskedPosition <= p) {
    firstUnmaskedPosition = seekNext(buffer, firstUnmaskedPosition);
}
```

**Comment by RobinHerbots** [comment]
Hi Gyrbo, 

I will have a look at it.

best regards, 

Robin

**Comment by RobinHerbots** [comment]
Fixed in version 1.0.25.

best regards, 

Robin

**Comment by RobinHerbots** [comment]
http://jsfiddle.net/SRTJj/1/
